src/ast/passes/attachpoint_passes.cpp

```diff
@@ -730,6 +750,66 @@ AttachPointParser::State AttachPointParser::uprobe_parser(bool allow_offset,
     ap_->target = parts_[1];
   }
 
+  // Statement mapped attach point
+  bool stmt_mapped = (!parts_[2].empty() && parts_[2].at(0) == '@');
+
+  // Handle uprobe:/bin/foo@bar.c:100[:1] case
+  if (stmt_mapped) {
+    if (!allow_stmt_map) {
+      errs_ << "Statement to address mapping not allowed" << std::endl;
```

Comment:
The word "Statement" here and in the code feels a little ambiguous (even though it may be correct). What about "source location"?